mlir_graphblas/src/test/GraphBLAS/structuralize_apply.mlir

```diff
@@ -361,31 +361,43 @@ module {
         return %answer : tensor<8xi8, #SparseVec64>
     }
 
-// CHECK-LABEL:   func @apply_minv_int_matrix(
+// CHECK-LABEL:   builtin.func @apply_minv_int_matrix(
```

Comment:
We can drop the `builtin.`. It will be dropped formally in the future, but even while we're in this weird state with the current MLIR build, Filecheck will still match starting from an arbitrary point in the line. So dropping `builtin.` will work today and in the future.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/35a3d57cb129d01978886379d0c1aab19388680d implements this suggestion.